### PR TITLE
panel elements release notes

### PR DESCRIPTION
**Pull request by ryangec**
release notes: 2 new panel elements

### CODE REVIEW

File: guides/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### New: Blocks Panel Builder Elements
```
Comment (imnotashrimp) [inline guides/release-notes.md:10]:
```suggestion
### New: Wix Blocks Panel Color and Font Picker Elements
```

- "Blocks" on its own isn't a term.
- Be specific about the elements being released

File: guides/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,15 @@ We love hearing your feedback! You can help us improve the docs by reporting any
...
+
+>**Note:** Wix Blocks is currently a Beta version, open to a limited number of users.
```
Comment (imnotashrimp) [inline guides/release-notes.md]: remove

```suggestion
```

File: guides/release-notes.md [hunk 3]
```diff
@@ -7,6 +7,15 @@ We love hearing your feedback! You can help us improve the docs by reporting any
...
+
+The following elements can be added to [Blocks Custom Panels](https://support.wix.com/en/article/wix-blocks-about-panels#custom-panels) and managed in the [Blocks Panel Code](https://support.wix.com/en/article/wix-blocks-adding-code-to-your-custom-panels):
```
Comment (imnotashrimp) [inline guides/release-notes.md:12]: Need a quick sentence or two (or three) describing the new elements - What's a color picker? What's a font and color picker?

**Review by roysha-wix — APPROVED** [review]
👌

**Pull request merged**